tests/persistence/test_persistence.py

```diff
@@ -0,0 +1,42 @@
+import six
+if six.PY2:
+    from backports import tempfile
+else:
+    import tempfile
+
+import pytest as pt
+import os
+from ditto.readers.opendss.read import Reader as Reader_opendss
+from ditto.readers.cyme.read import Reader as Reader_cyme
+from ditto.writers.json.write import Writer
+from ditto.store import Store
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+test_list = os.walk('data')
+for (dirpath, dirname, files) in test_list:
+    if files !=[]:
+        reader_type = dirpath.split('\\')[2]
+        m = Store()
+        if reader_type == 'opendss':
+            reader = Reader_opendss(master_file = os.path.join('..',dirpath,'master.dss'), buscoordinates_file = os.path.join('..',dirpath,'buscoord.dss'))
+        elif reader_type == 'cyme':
+            reader = Reader_cyme(data_folder_path=os.path.join('..',dirpath))
+        else:
+            #Update with other tests if they get added to the persistence tests
+            continue
```

Comment:
Raise `NotImplementedError` here instead.

Reply:
Not sure but if you do that you'll have your test always fail as long as you don't implement the missing format, right?

Reply:
I've left this as-is for now given that there are folders with other formats that may cause this test to fail if we raise an a `NotImplementedError` here.